README.md

```diff
@@ -243,6 +244,28 @@ See gif in action:
 The built-in multi-cursor support may not meet your needs. Please refer to the plugin
 [vscode-multi-cursor.nvim](https://github.com/vscode-neovim/vscode-multi-cursor.nvim) for more multi-cursor features
 
+### Remote development
+
+We intend to use vscode-neovim as a UI extension, running locally instead of installing and configuring nvim in a remote
+environment. If you prefer to use nvim in a remote environment, you can set the following in your VSCode
```

Comment:
It might be worth noting this isn't a requirement. I've never done it on any server I've connected to remotely

Reply:
Are you sure that the extension is running on remote extension host?

Reply:
I'm not, no, but this paragraph makes it sound like these settings are a requirement for using the extension when remotely connecting

Reply:
Adjusted a little